src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -685,6 +675,47 @@ private void getCanvasResponse(HttpServletRequest request,
 		out.flush();
 	}
 
+	private String extractNextLink(HttpResponse canvasResponse, String linkValueString) {
+		String linkValueNext = null;
+		String searchPhrase = "rel=\"next\"";
+		M_log.debug("SEARCH PHRASE: " + searchPhrase);
+		HashMap<String, String> links = new HashMap<String, String>();
+		Header[] headers = canvasResponse.getAllHeaders();
+		for (Header header : headers) {
+			M_log.debug("Key : " + header.getName() 
+					+ " ,Value : " + header.getValue());
+			if(header.getName().equals("Link")){
+				linkValueString = header.getValue();
+				String[] strings = linkValueString.split(",");
+				for(String string : strings){
```

Comment:
I think it `strings` and `string` and have better names to reflect what it is holding